db/seeds.rb

```diff
@@ -59,6 +61,24 @@
 bridge_builders2 = Tenant.find(2)
 bridge_builders3 = Tenant.find(3)
 
+
+# user as a tenant
+tenant_user = User.create!(
+             :username => "Jwan622",
```

Comment:
Indent the first parameter one step more than the previous line.<br>Use the new Ruby 1.9 hash syntax.